.github/workflows/pr-assign.yml

```diff
@@ -0,0 +1,19 @@
+name: Auto Request Review
+
+on:
+  pull_request:
+    # TODO remove synchronize event
```

Comment:
We won't need "synchronize" once this is g2g. Only here for testing so it runs each time the PR has code pushed to it